resources/python/install_binary_linux.sh

```diff
@@ -22,7 +22,7 @@ rm -rf $bin_dir
 mkdir $bin_dir
 
 # Install packages
-python build/pip install --target $bin_dir --no-deps -r $requirements
```

Comment:
Why did we originally have pip used in the build folder?

Also, can you make these scripts run in "set -e" please? (and equivalent for PowerShell) Because we want to make sure they stop at the first error since we are automating everything.

Reply:
I'm not familiar with the older syntax (`python build/pip...`), but it seems it was implemented some years ago.